docs/source/install/config/config.rst

```diff
@@ -6,7 +6,7 @@ serving static content of WebUI,
 and running st2auth and st2api as WSGI apps via gunicorn/uwsgi. |st2| nginx configurations
 can be found at ``/etc/nginx/sites-enabled/st2*.conf``.
 
-``st2auth`` and ``st2api`` can also run using a built-in simple Python server. This is used for development and strongly discouraged for any production. Be aware that some settings in /etc/st2.conf are only effective when running in development mode, and don't apply when running under WSGI servers. Refer to the comments in
+Be aware that some settings in /etc/st2.conf are only effective when running in development mode, and don't apply when running under WSGI servers. Refer to the comments in
```

Comment:
Should this also be /etc/st2/st2.conf?

And should make it clear that this only applies to listen ip and port settings for st2 auth and st2 api.